### PR TITLE
crash on empty deps.js file

### PR DESCRIPTION
**Pull request by rifler**

### CODE REVIEW

**Comment by skad0** [comment]
I think empty deps.js files should be skipped not in parser, but in reader. 
@zxqfox

File: lib/formats/deps.js/parser.js [hunk 1]
```diff
@@ -29,6 +29,11 @@ module.exports = function parse(depsData) {
         const data = [].concat(record.data);
 
         data.forEach(dep => {
+            if (!dep) {
+                console.warn(`Empty deps.js block in ${record.path}. Ignore it.`);
```
Comment (skad0) [inline lib/formats/deps.js/parser.js]: At first look, you are testing every object came in deps. For example, if we got smth like this:
```javascript
shouldDeps: [ false, false ]
```
We will get two warn messages, isn't it?

**Comment by zxqfox** [comment]
I'd say we should check it and warn about in linter. Here we dont actually need warns.

Also there was an idea to use json scheme validation in dev mode to report cases like these.

**Comment by rifler** [comment]
ок, i'll close this pr

**Pull request closed without merging**